aiida_quantumespresso/workflows/pdos.py

```diff
@@ -452,10 +467,14 @@ def _generate_dos_inputs(self):
         dos_inputs = AttributeDict(self.exposed_inputs(DosCalculation, 'dos'))
         dos_inputs.parent_folder = self.ctx.nscf_parent_folder
         dos_parameters = self.inputs.dos.parameters.get_dict()
+        fermi_energy_range = self.inputs.fermi_energy_range
```

Comment:
I think you forgot to update it here

Reply:
Oh yes, I am sorry. Now there is no `fermi_energy_range` left